chat-plugins/mafia.js

```diff
@@ -1278,9 +1285,12 @@ exports.commands = {
 			let targetUser = this.targetUser;
 			if (!targetUser || !targetUser.connected) return this.errorReply(`The user "${this.targetUsername}" was not found.`);
 			if (!room.users[targetUser.userid]) return this.errorReply(`${targetUser.name} is not in this room, and cannot be hosted.`);
-			if (Object.keys(room.game.players).includes(targetUser.userid) || Object.keys(room.game.dead).includes(targetUser.userid)) return this.errorReply(`You cannot subhost to a user in the game or a user that could be revived.`);
 			if (room.game.hostid === targetUser.userid) return this.errorReply(`${targetUser.name} is already the host.`);
-
+			if (targetUser.userid in room.game.players) return this.errorReply(`You cannot subhost to a user in the game.`);
+			if (targetUser.userid in room.game.dead) {
+				if (cmd !== 'forcesubhost') return this.errorReply(`${targetUser.name} could potentially be revived. To subhost anyway, use /mafia forcesubhost ${target}.`);
+				delete room.game.dead[targetUser.userid];
```

Comment:
You also need to run .destroy() on that before deleting it. (RoomGamePlayer.destroy() to be specific)

Reply:
nvm im dumb